app/javascript/app/data/SEO.js

```diff
@@ -1,16 +1,23 @@
 export const TITLE = 'Climate Watch: Data for Climate Action';
 export const HOME_PAGE =
-  'Climate Watch is an open online platform designed to empower users with the climate data, visualizations and resources they need to gather insights on national and global progress on climate change, sustainable development, and help advance the goals of the Paris Agreement.';
+  'Climate Watch is an open online platform designed to empower users with the climate data, ' +
```

Comment:
Same, template literals make it easier to update the text

Reply:
same as above, eslint warnings